packages/prisma/seed-insights.ts

```diff
@@ -83,65 +85,37 @@ const shuffle = (
 
 const prisma = new PrismaClient();
 async function main() {
-  // First find if not then create everything
-  let insightsAdmin = await prisma.user.findFirst({
+  // First find the organization we want to add insights to
+  const organization = await prisma.team.findFirst({
     where: {
-      email: "insights@example.com",
+      slug: "acme",
+      organizationSettings: {
+        isOrganizationVerified: true,
+        orgAutoAcceptEmail: "acme.com",
+        isAdminAPIEnabled: true,
+      },
     },
-  });
-
-  if (!insightsAdmin) {
```

Comment:
Note we may need to add this user back. I'm pretty sure it is used in a e2e test. Will check just wanted to get this out for early reviews